src/soep_preparation/merge_data/helper.py

```diff
@@ -0,0 +1,271 @@
+"""Helper functions to merge variables to dataset."""
+
+from difflib import get_close_matches
+
+import pandas as pd
+
+from soep_preparation.utilities.error_handling import (
+    fail_if_empty,
+    fail_if_input_has_invalid_type,
+)
+
+ID_VARIABLES = ["hh_id", "hh_id_original", "p_id", "survey_year"]
```

Comment:
Should be in config, I guess? Why do we need both `hh_id` and `hh_id_original` ?